apps/hmc/hmc-hmi-inbound-adapter/perftest.yaml

```diff
@@ -5,4 +5,11 @@ metadata:
 spec:
   values:
     java:
-      memoryRequests: '1Gi'
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '1536Mi'
```

Comment:
You'll need to increase the memory scaling threshold to 120%, otherwise it'll immediately scale up to 3 pods per cluster based on current utilisation.